data/theranamuskan.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Rana Muskan",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Contributor, Web Developer, Flutter Developer, Youtuber",
+  "avatar": "https://github.com/theranamuskan.png",
```

Comment:
As of our latest update, we no longer need the `avatar` key. So please remove this form your file. 
```suggestion
```